src/unix/core.c

```diff
@@ -823,7 +823,7 @@ void uv__io_init(uv__io_t* w, uv__io_cb cb, int fd) {
 
 
 void uv__io_start(uv_loop_t* loop, uv__io_t* w, unsigned int events) {
-  assert(0 == (events & ~(POLLIN | POLLOUT | UV__POLLRDHUP)));
+  assert(0 == (events & ~(POLLIN | POLLOUT | UV__POLLRDHUP | POLLPRI)));
```

Comment:
It has happened before: `POLLRDHUP` was not defined in some platforms i.e `aix`.